vraptor-core/src/main/java/br/com/caelum/vraptor/validator/DefaultValidationException.java

```diff
@@ -27,7 +29,7 @@
  */
 @br.com.caelum.vraptor.validator.annotation.ValidationException
 @Vetoed
-public class DefaultValidationException extends RuntimeException {
+public class DefaultValidationException extends VRaptorException {
```

Comment:
should this extend from ValidationException instead? Why do we have two exceptions for this?